docs/gc.md

```diff
@@ -31,35 +31,57 @@ The Allocator assigns areas of the Java heap for Java objects. See [Memory alloc
 
-All OpenJ9 GC policies support compressed references on 64-bit platforms, which compresses heap pointers to 32 bits if the total heap size does not exceed 63 GB. Applications that require more heap space can forgo compressed pointers and select any heap size within the bounds imposed by the operating system and available system RAM.
+A GC cycle is a repeatable process that involves a set of GC operations. These operations process all or parts of the Java heap to complete a discrete function. For example, a *mark* operation traces all objects in the heap to determine which ones are reachable. A *sweep* operation runs to clear away unreachable objects. Together, a *mark* and *sweep*  operation are capable of reclaiming used memory as part of a GC cycle.
+
+A GC operation might complete in one step, or it might involve phases. For example, a *mark* operation consists of the following 3 phases:
+
+- *Initial phase:* Identify all the root objects by running a root scan.
+- *Main phase:* Trace references from each root object to other objects in the heap. Reachable objects are marked.
+- *Final phase:* Run a further root scan to update references, followed by further operations, such as card cleaning.
+
+GC policies use different GC cycles to manage different aspects of the heap. For example, the `gencon` policy runs a partial GC cycle on the *nursery* area of the heap to complete a *scavenge* operation. At other times, `gencon` also runs a global GC on the entire Java heap to complete a *mark* and *sweep* (and optionally *mark* and *compact*) operation.  
+
+GC cycles are divided into increments that run over a period of time to reduce maximum pause times. These increments might involve *stop-the-world* (STW) pauses that must halt application threads to give certain GC operations exclusive access to the Java heap. Alternatively, increments might include GC operations that can run concurrently with application processing.
+
+The following table shows the heap configuration and the GC cycles and operations used by different policies:
+
+| Policy         | Heap configuration |  GC cycles / operations |
+|----------------|--------------------|-------------------------|
+|`gencon`        | Two areas: *nursery* and *tenure* <br> Two generations of objects: new/older |Global GC cycle: concurrent *mark-sweep* (and optionally *mark-compact*) operation <br> Partial GC cycle: STW *scavenge* operation or concurrent *scavenge* operation (if `-Xgc:concurrentScavenge` is set) |
+|`balanced`      | Multiple regions of equal size <br> Multiple generations: 24 |Global GC cycle: incremental concurrent *mark* operation (*global mark phase*), followed by a global STW *sweep* operation <br> Partial GC cycle: STW *scavenge* operation  or STW *mark-compact* operation |
```

Comment:
I would not use 'scavenge' in Balanced. it's a specific name for Gencon. 
You can use 'copy forward'